CondFormats/SiPixelTransient/src/SiPixelUtils.cc

```diff
@@ -0,0 +1,90 @@
+#ifndef SI_PIXEL_TEMPLATE_STANDALONE
```

Comment:
Perhaps, if you include the implementation in the header file you can avoid this specialization with ```#ifndef``` etc.

Reply:
Sorry I am not sure I understand, you are saying it would be preferable to have the implementation of generic_position_formula in interface/SiPixelUtils.h removing the need for src/SiPixelUtils.cc ? I thought it was desired to have function specifications and only simple functions actually implemented in the header file. Or am I mis-understanding?

Reply:
It was just a suggestion in case you wanted to avoid the ```#ifndef``` directive to specialize for the standalone case: it can work either way, anyhow